testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RefreshTokenTest.java

```diff
@@ -1132,6 +1285,79 @@ public void refreshTokenUserSessionMaxLifespanWithRememberMe() throws Exception
         }
     }
 
+    @Test
+    public void refreshTokenClientSessionMaxLifespan() throws Exception {
+        RealmResource realm = adminClient.realm("test");
+        RealmRepresentation rep = realm.toRepresentation();
+        Integer originalSsoSessionMaxLifespan = rep.getSsoSessionMaxLifespan();
+
+        ClientResource client = ApiUtil.findClientByClientId(adminClient.realm("test"), "test-app");
+        ClientRepresentation clientRepresentation = client.toRepresentation();
+
+        getTestingClient().testing().setTestingInfinispanTimeService();
+
+        try {
+            rep.setSsoSessionMaxLifespan(10);
```

Comment:
Is it possible to use bigger values for this test? For example 100 instead of 10 and also multiply all subsequent calls to `setTimeOffset` by 10? AFAIR in some slow environments, small times/timeout values can cause headaches and test instabilities. As in some environments, this test can easily take 3 seconds or more, which could result in the test failure.

Reply:
Yes, no problem, this test was taken from one of the previous PRs. I have used *100. So now the times are 1000, 500, ans so on and so forth.